site/routes/definitions.js

```diff
@@ -56,6 +69,16 @@ export const routeDefinitions : Array<RouteDefinition> = [
     gen: state => state.events.map(({ startDateTime: { date, month, year }, slug }) => ({ date, month, year, slug })),
   },
   {
+    title: ({ category }) => 'Meet our team' + (category !== 'everyone' ? ` (${category})` : ''),
+    key: 'badgers',
+    route: 'about-us/people/{category?}/{page?}',
+    defaults: { category: 'everyone', page: 1 },
+    urlEncode: (_, key, val) => (val === 'ux & design' ? 'ux-design' : encodeURIComponent(val)),
+    urlDecode: (_, key, val) => (val === 'ux-design' ? 'ux & design' : decodeURIComponent(val)),
```

Comment:
@asavin @JoeStanton We're not hard-coding a list of valid categories. We're just returning badgers from badger brain with their associated categories. Then, on the fetcher side, we're working out the categories by looping through the badgers to generate a unique list of those assigned. This means that any category assigned in Prismic, e.g., 'example category' will appear as a filter in the website - is this desirable? It also makes it difficult to create slugs because the category list is dynamic.

Shall we switch to a hard-coded list of categories and their slugs returned from graphql?

We could still filter this list on the fetcher side so that only those categories with badgers attached actually appear.